buildSrc/src/main/java/org/elasticsearch/gradle/test/RestTestBasePlugin.java

```diff
@@ -41,7 +41,7 @@ public void apply(Project project) {
             NamedDomainObjectContainer<ElasticsearchCluster> testClusters = (NamedDomainObjectContainer<ElasticsearchCluster>) project
                 .getExtensions()
                 .getByName(TestClustersPlugin.EXTENSION_NAME);
-            ElasticsearchCluster cluster = testClusters.create(restIntegTestTask.getName());
+            ElasticsearchCluster cluster = testClusters.maybeCreate(restIntegTestTask.getName());
```

Comment:
When will this have not already been created? Shouldn't the java/yaml plugins have already created it?

Reply:
We still have a quite a few QA tests that use the StandaloneRestTest (instead of yamlRestTest or javaRestTest) those tests generally are more customized tests with specific setup/creation in the build.gradle files via .create.